v2/internal/testcommon/test_recorder.go

```diff
@@ -21,7 +21,7 @@ func createTestRecorder(
 	cfg config.Values,
 	recordReplay bool,
 	log logr.Logger,
-) (vcr.Interface, error) {
+	customRedactMap map[string]string) (vcr.Interface, error) {
```

Comment:
```suggestion
	redactions map[string]string) (vcr.Interface, error) {
```

or maybe just `redact`?